Makefile

```
@@ -79,13 +79,16 @@ GINKGO_FOCUS ?=
 GINKGO_SKIP ?=
 GINKGO_NODES ?= 1
 GINKGO_TIMEOUT ?= 3h
-E2E_CONF_FILE ?= $(abspath test/e2e/config/vsphere-dev.yaml)
+E2E_CONF_FILE ?= $(abspath test/e2e/config/vsphere.yaml)
+E2E_CONF_OVERRIDES ?= $(abspath test/e2e/config/byo-overrides.yaml)
+E2E_CAPV_MODE ?= govimomi
```

```suggestion
E2E_CAPV_MODE ?= govmomi
```

Probably also in other places. It's just pronounced "vi" because of the "v" :)